app/services/exports/export_distributions_csv_service.rb

```diff
@@ -84,6 +92,11 @@ def base_table
       }
     end
 
+    # if filtered based on an item, change the column accordingly
+    def base_item_header_col_name
+      @filtered_item_id.present? ? "Total Number of Items" : "Total Items"
+    end
```

Comment:
I think to make this a bit more user friendly could you actually change "Items" to the item name. For example:
```

    # if filtered based on an item, change the column accordingly
    def base_item_header_col_name
      @filtered_item_id.present? ? "Total Number of #{filtered_item.name}" : "Total Items"
    end

    def filtered_item
      @filtered_item ||= Item.find(@filtered_item_id)
    end
```

Reply:
makes sense, made the changes.